docs/faq.md

```diff
@@ -77,6 +77,37 @@ sudo rm /var/db/dhcpd_leases
 
 And no worries, this file will be re-created on the next `tart run`.
 
+## Unsupported DHCP client identifiers
+
+Due to the limitations of the macOS built-in DHCP server, `tart ip` is unable to correctly report the IP addresses for VMs using DHCP client identifiers that are not based on VMs link-layer addresses (MAC addresses).
+
+One notorious example of this is Ubuntu, using [DUID-EN](https://metebalci.com/blog/a-note-on-dhcpv6-duid-and-prefix-delegation#duid-types) identifier by default on new versions, which results in the `/var/db/dhcpd_leases` entry for Ubuntu appearing as follows:
+
+```ini
+{
+    name=ubuntu
+    ip_address=192.168.64.14
+    hw_address=ff,f1:f5:dd:7f:0:2:0:0:ab:11:cb:fb:30:b0:97:b6:3a:67
+    identifier=ff,f1:f5:dd:7f:0:2:0:0:ab:11:cb:fb:30:b0:97:b6:3a:67
+    lease=0x678e2ce7
+}
+```
+
+Because the macOS built-in DHCP server overwrites the `hw_address` with the `identifier`, it leaves no information about the VM's MAC address to the `tart ip`, which relies on this file to determine the IP address associated with the VM's MAC address.
```

Comment:
It seems like tart ip depends on /var/db/dhcpd_leases, which cannot work for bridged network, since the builtin macOS dhcps server is not the the loop.

It would be nice to add a sentence making this more clear (tart looks up the vm ip address in /var/db/dhcpd_leases...), and add a section about bridged network.

Reply:
See https://github.com/cirruslabs/tart/pull/1009/commits/98ba3675c006e542aca1b4218afbe867d8d40697.